greedy_methods/fractional_knapsack.py

```diff
@@ -6,6 +6,10 @@ def frac_knapsack(vl, wt, w, n):
     """
     >>> frac_knapsack([60, 100, 120], [10, 20, 30], 50, 3)
     240.0
+    >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 10, 4)
+    105.0
+    >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 8, 4)
+    95.0
```

Comment:
```suggestion
    95.0
    >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6], 8, 4)
    ?
    >>> frac_knapsack([10, 40, 30], [5, 4, 6, 3], 8, 4)
    ?
    >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 0, 4)
    ?
    >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 8, 0)
    ?
    >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], -8, 4)
    ?
    >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 8, -4)
    ?
    >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 800, 4)
    ?
    >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 8, 400)
    ?
```

Reply:
I have added the above doctests to the code.